src/se/edu/inclass/task/Task.py

```diff
@@ -0,0 +1,6 @@
+class Task:
```

Comment:
can make it abc?

Reply:
the current code structure instantiates this class task (i think in the DataManager) - which makes it a bit difficult to change this class Task to abc